Rest/Controller/ContentController.php

```diff
@@ -278,4 +292,53 @@ private function getRelation($content, $language)
 
         return false;
     }
+
+    /**
+     * Returns author of the content.
+     *
+     * @param Content $contentValue
+     * @param ContentType $contentType
+     *
+     * @return string
+     */
+    private function getAuthor(Content $contentValue, ContentType $contentType)
+    {
+        $author = $contentValue->getFieldValue($this->getFieldIdentifier('author', $contentType));
+
+        if (null === $author) {
+            $user = $this->userService->loadUser($contentValue->contentInfo->ownerId);
+            $author = trim($user->getFieldValue('first_name').' '.$user->getFieldValue('last_name'));
```

Comment:
I would use $user->getName() (I did not check if that method is available). It is more flexible. Usually name is built on first_name and last_name.